src/processor/operator/shortest_path.cpp

```diff
@@ -0,0 +1,100 @@
+#include "processor/operator/shortest_path.h"
+
+#include "iostream"
+
+using namespace std;
+
+namespace kuzu {
+namespace processor {
+
+bool ShortestPathAdjList::getNextTuplesInternal() {
```

Comment:
This function is not well organized. Can you separate the logic of reading a pair of src and dest node ids, and the logic of computing shortest path between them?
So, you will have a computeShortestPath function, which takes in source and dest node id, and return a path (you can represent the path in any data type you want for now, e.g., string; vector of int64 with mixed node and rel ids; vector of steps, each contains a source node id, rel id, and dest node id)

Then, getNextTuplesInternal will be responsible for reading src and dest node ids from `srcValueVector` and `destValueVector`, and them call computeShortestPath, and finally print the result for it.

Reply:
I've changed the format of the logic, currently it is like this:
<pre>
while(true):
  computeShortestPath;
  if(child->getNextTuples())
    getSrcNode and getDestNode;
</pre>